docs/data/stashcache/install-cache.md

```diff
@@ -5,100 +5,130 @@ This document describes how to install a StashCache cache service.  This service
 network to cache data frequently used on the OSG, reducing data transfer over the wide-area network and
 decreasing access latency.
 
-!!! note
-    The _cache_ service must be registered with the OSG if it is to be used by clients. You may start 
-    registration procedure prior to finishing the installation by [using this link](#registering-the-cache) 
-    along with the basic information like:
-
-    * Resource name and hostname.
-    * Administrative and security contact.
 
-## Installation prerequisites for the Cache
+Installation Prerequisites for the Cache
+----------------------------------------
 
-Before starting the installation process, consider the following mandatory points:
+Before starting the installation process, consider the following requirements:
 
-* __Operating system:__ Only RHEL 7 and compatible operating systems are supported.
+* __Operating system:__ A RHEL 7 or compatible operating systems.
 * __User IDs:__ If they do not exist already, the installation will create the Linux user IDs `condor` and
   `xrootd`
-* __Host certificate:__ The StashCache server uses a host certificate to advertise to a central collector.
+* __Host certificate:__ Optional, required for authenticated StashCache.
   The [host certificate documentation](/security/host-certs.md) provides more information on setting up host
   certificates.
-* __Network ports:__ The cache service requires inbound ports, one for the xrootd protocol, one for
-  HTTP and, if authenticated StashCache is used, one for HTTPS.  The defaults are:
-    * TCP port 1094 for xrootd.
-    * TCP port 8000 for HTTP.
-    * TCP port 8443 for HTTPS (optional).
-* __Hardware requirements:__ We recommend that a StashCache server has at least 10Gbps connectivity, 1TB of
+* __Network ports:__ The cache service requires the following ports open:
+    * Inbound TCP port 1094 for file access via the XRootD protocol
+    * Inbound TCP port 8000 for file access via HTTP
+    * Inbound TCP port 8443 for authenticated file access via HTTPS (optional)
+    * Outbound UDP port 9930 for reporting to `xrd-report.osgstorage.org` and `xrd-mon.osgstorage.org` for monitoring
+* __Hardware requirements:__ We recommend that a StashCache cache has at least 10Gbps connectivity, 1TB of
```

Comment:
If it were just the "OSG Stash Federation" then we could use "Stash Cache" but we don't so when we say the proper name it has to be a "StashCache cache." We can get away with calling the RPM and the files `stash-cache` because it's an abbreviation.

Reply:
We are bad at naming things :(. Since this document is already clearly about Stash Cache, how about we drop "StashCache" from this sentence?

Reply:
Sure.